python/tskit/trees.py

```diff
@@ -8717,11 +8686,7 @@ def tjd_func(sample_set_sizes, flattened, **kwargs):
             g = np.array([np.sum(1 / np.arange(1, nn) ** 2) for nn in n])
             with np.errstate(invalid="ignore", divide="ignore"):
                 a = (n + 1) / (3 * (n - 1) * h) - 1 / h**2
-                b = (
-                    2 * (n**2 + n + 3) / (9 * n * (n - 1))
-                    - (n + 2) / (h * n)
-                    + g / h**2
-                )
+                b = 2 * (n**2 + n + 3) / (9 * n * (n - 1)) - (n + 2) / (h * n) + g / h**2
```

Comment:
I don't really like this change - the previous version was written out on multiple lines so it's easier to parse by eye. However, most of these many lines -> one line changes I do like, so maybe it's okay?

Reply:
In this one case you could #noqa the line, I guess?